app/admin/page.tsx

```diff
@@ -37,6 +38,36 @@ function getUsers(props: {pageIndex:number}): Promise<User[]> {
     })
 }
 
+function deleteSandbox(row: Row<any>) {
+  let user:User = row.original
+
+  if (!user.task_arn){
+    toast({
+      title: "Error",
+      description: "No task_arn found",
+    })
+    return
+  }
+
+  fetch(`/api/db/${encodeURIComponent(user.task_arn)}`, {
```

Comment:
why not using async/await?

Reply:
just because this is how I've done it in other places